controllers/hibp.js

```diff
@@ -12,7 +12,16 @@ async function notify (req, res) {
   const subscribers = await DB.getSubscribersByHashes(hashes);
 
   const reqBreachName = req.body.breachName.toLowerCase();
-  const breach = req.app.locals.breaches.find(breach => breach.Name.toLowerCase() === reqBreachName);
+  let breach = req.app.locals.breaches.find(breach => breach.Name.toLowerCase() === reqBreachName);
+  if (!breach) {
+    // If breach isn't found, try to reload breaches from HIBP
+    await HIBP.loadBreachesIntoApp(req.app);
+    breach = req.app.locals.breaches.find(breach => breach.Name.toLowerCase() === reqBreachName);
+    if (!breach) {
+      // If breach *still* isn't found, we have a real error
+      throw new Error("Unrecognized breach: " + reqBreachName);
+    }
+  }
```

Comment:
On L10-12 above we're doing something like:

```js
const hashes = req.body.hashSuffixes.map(suffix => req.body.hashPrefix + suffix);
const subscribers = await DB.getSubscribersByHashes(hashes);
```

Should that move down here after the `throw new Error()`? Otherwise, it seems like we're doing all the heavy lifting of fetching a bunch of subscribers based on hashes for a possibly invalid breach.